rocketpy/simulation/dispersion.py

```diff
@@ -22,111 +22,109 @@
-    Dispersion.outputs_log : list
-        List in which each item is a line of the output_file.
-    Dispersion.errors_log : list
-        List in which each item is a line of the error_file.
-    Dispersion.num_of_loaded_sims : int
+    environment : StochasticEnvironment
+        The stochastic environment object to be iterated over.
+    rocket : StochasticRocket
+        The stochastic rocket object to be iterated over.
+    flight : StochasticFlight
+        The stochastic flight object to be iterated over.
+    export_list : list
+        The list of variables to export. If None, the default list will
+        be used. Default is None. # TODO: improve docs to explain the
+        default list, and what can be exported.
+    inputs_log : list
+        List of dictionaries with the inputs used in each simulation.
+    outputs_log : list
+        List of dictionaries with the outputs of each simulation.
+    errors_log : list
+        List of dictionaries with the errors of each simulation.
+    num_of_loaded_sims : int
         Number of simulations loaded from output_file being currently used.
-    Dispersion.results : dict
-        A dictionary containing all the output parameters saved from the flight
-        simulations.
-    Dispersion.processed_results : dict
-        Dictionary containing (mean, std. dev.) for each parameter available
-        in the dispersion dictionary.
+    results : dict
+        Dispersion results organized in a dictionary where the keys are the
+        names of the saved attributes, and the values are a list with all the
+        result number of the respective attribute
+    processed_results : dict
+        Creates a dictionary with the mean and standard deviation of each
+        parameter available in the results
+    prints : _DispersionPrints
+        Object with methods to print information about the dispersion
+        simulation.
+    plot : _DispersionPlots
+        Object with methods to plot information about the dispersion
+        simulation.
+    _inputs_dict : dict
+        Dictionary with the inputs of the last simulation.
+    _last_print_len : int
+        Used to print on the same line.
+    _input_file : str
+        String containing the filepath of the input file
+    _output_file : str
+        String containing the filepath of the output file
+    _error_file : str
+        String containing the filepath of the error file
+    _number_of_simulations : int
+        Number of simulations to be run, must be non-negative.
+    _iteration_count : int
+        Number of simulations already run.
+    _start_time : float
+        Time when the simulation started.
+    _start_cpu_time : float
+        CPU time when the simulation started.
+    _input_file : str
+        String containing the filepath of the input file
+    _output_file : str
+        String containing the filepath of the output file
+    _error_file : str
+        String containing the filepath of the error file
     """
```

Comment:
What is the point of documenting weakly private attribute? If you're going to document it, then why don't you just go and make them public? Also, all the descriptions here are totally predictable, meaning that if you don't write the docstring I can still understand it.

Reply:
I guess there is no point in having them. Will remove it